### PR TITLE
Create echo bot

### LINKED ISSUE #3: Create echo bot
In order to start developing properly, I would like to create simple echo-bot first.

### PR DESCRIPTION
**Pull request by dgreflex**
What's done:
- [x] Added .gitignore to ignore config.json containing token for telegram bot
- [x] Prepared a template for productive work on the echo bot
- [x] Added logging to collect bot operation logs
- [x] Added basic commands for the bot
- [x] Implemented echo function for resending messages
- [x] Added the main function that implements the main work of the telegram bot through the token

This PR closes #3

### CODE REVIEW

**Comment by dgreflex** [comment]
Closes #3

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+config.json
```
Comment (sanyavertolet) [inline .gitignore]: No newline at end of file.

File: echo-bot.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3.10
+
+import logging
+import json
+from telegram import Update, ForceReply
+from telegram.ext import (
```
Comment (sanyavertolet) [inline echo-bot.py:6]: Not a big fan of such imports but ok

File: echo-bot.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    Updater,
+    CommandHandler,
+    MessageHandler,
+    CallbackContext,
+    Filters,
+)
+
+logging.basicConfig(format="%(asctime)s - %(name)s - %(levelname)s - %(message)s", level=logging.DEBUG)
```
Comment (sanyavertolet) [inline echo-bot.py:14]: In release it should be logging.INFO, I suppose

**Review by sanyavertolet — CHANGES_REQUESTED** [review]
You should run pydocstyle on your code.

File: echo-bot.py (hunk 2 above)
Comment (dgreflex) [reply echo-bot.py:6]: This is the requirement of the "black" linter.

File: echo-bot.py (hunk 3 above)
Comment (dgreflex) [reply echo-bot.py:14]: Yes, of course, this is the initial version, so that we have something to start with.

**Review by sanyavertolet — APPROVED** [review]
lgtm

**Pull request merged**